### ISSUE TITLE
Add Stampatello Faceto

### ISSUE THREAD

**Issue by HaleyHalcyon** [google/fonts#8441]
**Font Project Git Repo URL:** https://gitlab.com/haleyhalcyon/stampatello-faceto

**Super short description of the Font Family:**

Felt pen handwriting font, supports Latin (incl. Vietnamese and some African languages), Cyrillic (incl. Ukrainian, Serbian), Greek (incl. polytonic), and Shavian.

I’m back for more after my first submission Orelega! The metrics for this font may need improvement, and the name could use shortening. Perhaps Spritulo (witticist; clever guy), keeping with Orelega’s Esperanto name?

**Requirements:**

By opening this issue, I confirm the project meets the following requirements:

- [ ] The entire font project is available in a Github repository (repo) and licensed under the [OFL](https://openfontlicense.org/open-font-license-official-text/)
- [x] The source files are available in the repo
- [x] I am the sole copyright author of the entire project, or, all other copyright authors have licensed their work to me under the OFL
- [x] There are no "Reserved Font Names" in the OFL license information, or in the project documentation of any known upstream projects. If there are RFNs, they are not used in whole or in part in this family name, or, I want to discuss how Google can work with my use of them.
- [x] The family name is unique according to [namecheck.fontdata.com](https://namecheck.fontdata.com/)
- [x] The font supports at least the Google Fonts 'Latin Core' glyphset from [github.com/googlefonts/glyphsets](https://github.com/googlefonts/glyphsets) ([direct link](https://github.com/googlefonts/glyphsets/blob/main/data/results/txt/nice-names/GF_Latin_Core.txt))
- [x] The repo has the [Google Fonts preferred upstream repo structure](https://googlefonts.github.io/gf-guide/upstream.html)
- [x] I have read, agree with, and comply with, the full [Google Fonts contributing requirements](https://googlefonts.github.io/gf-guide/index#pre-production-getting-your-fonts-ready-for-gf)
- [x] I will maintain the repository and participate in the onboarding process (addressing, solving, and responding to issues, merging pull requests, etc)

**Image:**

![](https://gitlab.com/haleyhalcyon/stampatello-faceto/-/raw/main/documentation/image2.png?ref_type=heads)

**Comment by emmamarichal** [comment]
Hello @HaleyHalcyon!

Thanks a lot for your submission, I add it to the submission to review.

However, we would need a Github repository so we can work on it properly (QA, exports) and to add it then on the Google Fonts repo.

We also don't usually work with fontforge format, would it be possible to add .ufos or .glyphs file in the sources folder?

I quickly opened the otf and saw that some glyphs are missing. In order to follow the specs, and if the project if accepted, you will have to add the following glyphs: `Wacute Wdieresis Wgrave idotaccent wacute wdieresis wgrave periodcentered.loclCAT periodcentered.loclCAT.case caroncomb.alt dotaccent hungarumlaut breve ring tilde ogonek `

We will let you know about the decision, thanks again!

**Comment by HaleyHalcyon** [comment]
> However, we would need a Github repository so we can work on it properly (QA, exports) and to add it then on the Google Fonts repo.

> We also don't usually work with fontforge format, would it be possible to add .ufos or .glyphs file in the sources folder?

I'm making a GitHub repo that is planned to use a .ufo3 file as the main project file, but upon exporting to UFO3 and re-importing, the feature file was not accepted as valid.

> Wacute Wdieresis Wgrave wacute wdieresis wgrave

Easy addition; already had the combining diacritics necessary for that.

> idotaccent

Isn't that just a normal lowercase i?

> periodcentered.loclCAT periodcentered.loclCAT.case

so U+013F and U+0140 are deprecated?

> caroncomb.alt

Is this the same as U+0313 combining comma above?

> dotaccent hungarumlaut breve ring tilde ogonek

U+0307, U+030B, U+0306, U+030A, U+0303, and U+0328 are already supported.

**Comment by HaleyHalcyon** [comment]
New repository on GitHub: https://github.com/HaleyHalcyon/spritulo

**Comment by emmamarichal** [comment]
Hi @HaleyHalcyon,

Thank you for the update, now I'm also able to open it with Glyphs, and it will be easier for me to engineer the font if needed :)
I'll take a closer look for the accents, but all the listed ones are in the GF glyphset and need to be in the font. Sometime the conversation of files creates some issues with the glyphs naming and unicode, I'll let you know!

Is there any reason for the changing name too? 

The font is on the submission to review milestone, we will let you know if it's accepted, thank you again!

**Comment by HaleyHalcyon** [comment]
> Thank you for the update, now I'm also able to open it with Glyphs, and it will be easier for me to engineer the font if needed :) I'll take a closer look for the accents, but all the listed ones are in the GF glyphset and need to be in the font. Sometime the conversation of files creates some issues with the glyphs naming and unicode, I'll let you know!

I'd like to know if you decide to normalize the glyph names to the Google Fonts standard, so I can avoid making unreconcilable changes.

> Is there any reason for the changing name too?

While using Stampatello Faceto, I realized that the font has letters that are too large for its font size. I shrank it to 80% size and renamed it to prevent accidental incompatibilities with my own projects that use the font.

> The font is on the submission to review milestone, we will let you know if it's accepted, thank you again!

Thank you for your consideration!

**Comment by emmamarichal** [comment]
> I'd like to know if you decide to normalize the glyph names to the Google Fonts standard, so I can avoid making unreconcilable changes.

Yes we will have to, you can refer to the Latin Core Glyphset: https://github.com/googlefonts/glyphsets/blob/main/data/results/glyphs/GF_Latin_Core.glyphs

> While using Stampatello Faceto, I realized that the font has letters that are too large for its font size. I shrank it to 80% size and renamed it to prevent accidental incompatibilities with my own projects that use the font.

alright, no problem!

**Comment by davelab6** [comment]
@HaleyHalcyon wow, this is an awesome project! I am happy to accept it. Great work on all the language support :) If you didn't already change the glyph names, I think its OK to keep your own glyph names, as long as the **character** support meets or exceeds the "GF Core" glyph sets for each script you've supported. 

I saw your note about FontForge bugs in your README, I believe FontForge is still being actively maintained so I do recommend trying to connect with the FF devs on their issue tracker about it. You could also try the new Fontra.xyz font editor. 

We will likely onboard this in Q3 :)

**Comment by HaleyHalcyon** [comment]
I’ve updated the font to expand Old Church Slavonic coverage
![](https://raw.githubusercontent.com/HaleyHalcyon/spritulo/main/examples/spritulo-desk.png)

**Comment by HaleyHalcyon** [comment]
Should I revise the spacing between the base character and the diacritics in Vietnamese?
![](https://github.com/HaleyHalcyon/spritulo/blob/main/examples/spritulo-chopsticks.png?raw=true)
